viz.yaml

```diff
@@ -6,19 +6,9 @@ info:
   publish-date: 2017-03-22
   path: /gages-through-ages
   analytics-id: UA-78530187-1
-  description: >-
```

Comment:
Looks like the second tab is the issue: 
![image](https://cloud.githubusercontent.com/assets/2349007/23805587/1f7c6eba-0584-11e7-8abf-000b2d111a63.png)